test/unit/link.ts

```diff
@@ -0,0 +1,128 @@
+import { Links } from '../../src';
+import { expect } from 'chai';
+
+describe('Links object', () => {
+
+  it('should instantiate', () => {
+
+    const links = new Links('http://base.example/');
+    expect(links).to.be.an.instanceof(Links);
+
+  });
+
+  it('should allow adding links', () => {
+
+    const links = new Links('http://base.example/');
+    links.add('rel', 'http://a.example/');
+    links.add({
+      rel: 'rel',
+      href: 'http://b.example/'
+    });
+
+    expect(links.getAll()).to.eql([
+      {
+        context: 'http://base.example/',
+        rel: 'rel',
+        href: 'http://a.example/',
+      },
+      {
+        context: 'http://base.example/',
```

Comment:
I may have missed this earlier, but what is the relationship between `context` and `href`?

Reply:
`context` is the base url basically.